go.sum

```diff
@@ -1,3 +1,5 @@
+4d63.com/gochecknoglobals v0.1.0 h1:zeZSRqj5yCg28tCkIV/z/lWbwvNm5qnKVS15PI8nhD0=
```

Comment:
How come all the new entries here? Have you tried to run go mod tidy?

Reply:
Resolved